.gitignore

```diff
@@ -151,14 +151,15 @@ $RECYCLE.BIN/
 # End of https://www.toptal.com/developers/gitignore/api/java,visualstudiocode,eclipse,windows,linux
 
 # Xyna specific stuff
-**/local.repository/*
-server/src/com/gip/xyna/xact/exceptions/*
-server/src/com/gip/xyna/xdev/exceptions/*
-server/src/com/gip/xyna/xfmg/exceptions/*
-server/src/com/gip/xyna/xnwh/exceptions/*
-server/src/com/gip/xyna/xmcp/exceptions/*
-server/src/com/gip/xyna/xprc/exceptions/*
-server/src/com/gip/xyna/exceptions/*
-**/xfcli/generated/*
+**/local.repository
+**/deploy
+**/lib
```

Comment:
Some components/modules contain checked in JARs (like [modules/xact/queue](https://github.com/GIP-SmartMercial/xyna-factory/tree/main/modules/xact/queue/common/lib)). What do we do about those?

Reply:
The problematic jars are mdm.jar, serviceDefinition.jar and serviceDefinition-javadoc.jar.

The long term goal is to remove them from the repository and generate them locally instead. There already is a mechanism to generate mdm.jar, but until that has matured and similar functions are in place to create the other jars, we should keep them in the repository.